docs/rules/use-baseline.md

```diff
@@ -104,6 +104,57 @@ This rule accepts an option object with the following properties:
 - `available` (default: `"widely"`)
     - change to `"newly"` to allow features that are at the Baseline newly available stage: features that have been supported on all core browsers for less than 30 months
     - set to a numeric baseline year, such as `2023`, to allow features that became Baseline newly available that year, or earlier
+- `allowAtRules` (default: `[]`) - Specify an array of at-rules that are allowed to be used.
+
```

Comment:
The general convention is to list all the options initially and then sub headings for each option under . [See this](https://eslint.org/docs/latest/rules/no-underscore-dangle#options) for reference